docs/control-plane.md

```diff
@@ -1,5 +1,7 @@
-# KFServing
-A KFService is a unit of model serving. Users may have many KFServices, including different definitions across dev, test, and prod environments, as well as across different model use cases. KFServices are defined using the [Kubernetes Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) standard. 
+# Control Plane Specification
+
+## KFService
+Control plane is responsible for the lifecycle of a KFService. A KFService is a unit of model serving. Users may have many KFServices, including different definitions across dev, test, and prod environments, as well as across different model use cases. KFServices are defined using the [Kubernetes Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) standard. 
```

Comment:
I'd prefer to not have this first sentence.

Reply:
Done